package.json

```diff
@@ -12,8 +12,8 @@
 		"node": "^16 || ^18 || >=20"
 	},
 	"scripts": {
-		"benchmarks": "node ./benchmarks/index.js",
-		"test": "npm run lint && npm run check-prettier && npm run compile-typescript && npm run test-unit",
+		"benchmarks": "node --max-old-space-size=8192 ./benchmarks/index.js",
```

Comment:
landing this in a separate PR makes sense 👍

Reply:
Okay